src/ufuncs.c

```diff
@@ -862,7 +862,18 @@ static void* get_z_data[1] = {GetZ};
 #endif
 static void* area_data[1] = {GEOSArea_r};
 static void* length_data[1] = {GEOSLength_r};
+
 #if GEOS_SINCE_3_6_0
+static int GetPrecision(void* context, void* a, double* b) {
+  // GEOS returns -1 on error; 0 indicates floating point precision
```

Comment:
doubtful if it indicates floating point precision (see previous remark) - if you agree please change also here